lib/helpers/PartialViewApiHelper.ts

```diff
@@ -8,45 +8,55 @@ export class PartialViewApiHelper {
   }
 
   async get(path: string) {
-    const response = await this.api.get(this.api.baseUrl + '/umbraco/management/api/v1/partial-view?path=' + path);
+    const response = await this.api.get(this.api.baseUrl + '/umbraco/management/api/v1/partial-view/' + encodeURIComponent(path));
     return await response.json();
   }
 
   async doesExist(path: string) {
-    const response = await this.api.get(this.api.baseUrl + '/umbraco/management/api/v1/partial-view?path=' + path);
+    const response = await this.api.get(this.api.baseUrl + '/umbraco/management/api/v1/partial-view/' + encodeURIComponent(path));
     return response.status() === 200;
   }
 
-  async create(name: string, content: string, parentPath: string = "") {
+  async create(name: string, content: string, parentPath?: string) {
+    const parentValue = parentPath ? {'path': parentPath} : null;
     const partialViewData = {
-      "name": name,
-      "content": content,
-      "parentPath": parentPath
+      'name': name,
+      'parent': parentValue,
+      'content': content
     };
     const response = await this.api.post(this.api.baseUrl + '/umbraco/management/api/v1/partial-view', partialViewData);
     // Returns the path of the created partialView
-    return response.headers().location.split("=").pop();
+    const partialViewPath = response.headers().location.split("/").pop() as string;
+    return decodeURIComponent(partialViewPath);
   }
 
-  async update(partialView) {
-    const partialViewData = {
-      "name": partialView.name,
-      "content": partialView.content,
-      "existingPath": partialView.path
-    };
-    return await this.api.put(this.api.baseUrl + '/umbraco/management/api/v1/partial-view', partialViewData);
+  async update(path: string, content: string) {
```

Comment:
I think the naming for updating content should be updateContent()
And updating name should be updateName()

Reply:
Makes sense